docs/src/main/paradox/general-architecture/index.md

```diff
@@ -0,0 +1,11 @@
+# General Architecture
+
+General documentation about how Guardian for Apache Kafka is architected lives here.
```

Comment:
Shouldn't be a trademark here?

Reply:
`Guardian for Apache Kafka` is the terminology that is trademark compliant if I understand your question correctly

Reply:
ok, good if so